Cork/Views/Maintenance/Sub-Views/Ready View.swift

```diff
@@ -80,7 +80,7 @@ struct MaintenanceReadyView: View {
                         print("Start")
                         maintenanceSteps = .maintenanceRunning
                     } label: {
-                        Text("Start Maintenance")
+                        Text("Start")
```

Comment:
This felt like repetition given the sheet tile is "Perform Brew Maintenance" (which arguably could be shortened to "Brew Maintenance")

Reply:
Good idea, it doesn't need that "Maintenance"